src/data_source/storage/sql/queries/explorer.rs

```diff
@@ -238,7 +238,8 @@ where
         };
 
         let block_height = transaction_target.get::<i64, _>("height") as usize;
-        let transaction_index = transaction_target.get::<Json<TransactionIndex<Types>>, _>("index");
+        let transaction_index =
+            transaction_target.get_unchecked::<Json<TransactionIndex<Types>>, _>("index");
```

Comment:
Why did this change to unchecked?

Reply:
so the type checking thing is broken for sqlite. I forgot the exact error but it was something like expected sql type `TEXT` found `INT` or viceversa

Reply:
Welp